ios/FluentUI/Vnext/List/List.swift

```diff
@@ -10,10 +10,19 @@ import SwiftUI
 @objc public class MSFListSectionState: NSObject, ObservableObject, Identifiable {
     public var id = UUID()
     @objc @Published public var cells: [MSFListCellState] = []
-    @objc @Published public var style: MSFHeaderFooterStyle = .headerPrimary
     @objc @Published public var title: String?
     @objc @Published public var backgroundColor: UIColor?
     @objc @Published public var hasDividers: Bool = false
+    @objc @Published public var style: MSFHeaderFooterStyle = .headerPrimary {
+        didSet {
+            if style != oldValue {
+                headerTokens.style = style
+                headerTokens.updateForCurrentTheme()
```

Comment:
Do we need this for published objects?

Reply:
This is because the Tokens class is not a subscriber of its own property. And the SwiftUI control does not use it to render its contents.